packages/ic-response-verification-tests/src/agent.rs

```diff
@@ -2,10 +2,11 @@ use anyhow::Result;
 use ic_agent::agent::http_transport::ReqwestHttpReplicaV2Transport;
 use ic_agent::Agent;
 
-pub fn create_agent(url: &str) -> Result<Agent> {
+pub async fn create_agent(url: &str) -> Result<Agent> {
```

Comment:
I also updated the rust tests to use the root key from the agent instead of passing it in from the script.
Retrospectively this feels like the obvious thing to do 😆